File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/api/BitgetApi.java
package com.bitget.openapi.api;

import com.bitget.openapi.dto.response.ResponseResult;
import retrofit2.Call;
import retrofit2.http.*;

import java.util.Map;

public interface BitgetApi {

    @GET
    Call<ResponseResult> sendGetRequest(@Url String url, @QueryMap Map<String, String> paramMap);

    @POST
    Call<ResponseResult> sendPostRequest(@Url String url, @Body Map<String, String> paramMap);
}

File: bitget-php-sdk-api/src/api/BitgetApi.php
<?php


namespace bitget\api;


use bitget\internal\BitgetApiClient;

class BitgetApi
{
    var BitgetApiClient $BitgetApiClient;

    public function __construct($BitgetApiClient)
    {
        $this->BitgetApiClient = $BitgetApiClient;
    }

    public function get($url, $params): string
    {
        return $this->BitgetApiClient->doGet($url, $params);
    }

    public function post($url, $params): string
    {
        return $this->BitgetApiClient->doPost($url, $params);
    }
}